jupyterlab_wipp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab_wipp",
-  "version": "0.3.1",
+  "version": "0.4.0",
```

Comment:
Is this a breaking change? Would the extension still work on the previous versions of JupyterLab? I think we are at a stable stage with this extension that you could bump the major version if necessary.

Reply:
You are right, it is a breaking change and it won't work in the previous version of JupyterLab. Perhaps we should change it to 1.0.0?